### DISCUSSION TITLE
How to realize the mask effect of modal dialog box?

### DISCUSSION THREAD

**Discussion by roxxxxssss** [MahApps/MahApps.Metro#4139] [Q&A] [answered]
I didn't find the effect I wanted in the official demo. The effect I want is shown in the picture. Who can help me? thank you.
![遮罩效果](https://user-images.githubusercontent.com/802371/123735576-101d0080-d8d2-11eb-9140-61d98034267a.png)

**Comment by roxxxxssss** [comment]
My new Build class : Controls:MetroWindow.

**Comment by timunie** [comment, marked as answer]
Hi @roxxxxssss 

what about using this lib: https://github.com/punker76/MahApps.Metro.SimpleChildWindow ?

I think with a custom style you should be able to create such a dialog. 

Happy coding 
Tim

**Reply by roxxxxssss** [reply]
yes,yes, Thank you very much.
